pkg/ssh/helper.go

```diff
@@ -56,6 +58,37 @@ func StdioClientWithUser(reader io.Reader, writer io.WriteCloser, user string, e
 	return StdioClientFromKeyBytesWithUser(nil, reader, writer, user, exitOnClose)
 }
 
+func TailscaleClientWithUser(ctx context.Context, ts tailscale.TSNet, serverAddress, user string, log log.Logger) (*ssh.Client, error) {
```

Comment:
this function's parameters are the same as the one below, we can remove one of them?

Reply:
yeah those helpers need rework in general, we also never really use the bytes passed in client defined above, it's always nil.